twitchio/ext/routines/__init__.py

```diff
@@ -392,6 +415,8 @@ async def _routine(self, *args, **kwargs) -> None:
                 sleep = max((start - datetime.datetime.now(datetime.timezone.utc)).total_seconds() + self._delta, 0)
 
             self._completed_loops += 1
+            # I believe this will provide a more realistic capture of what is happening
+            self.next_event_time = datetime.datetime.now() + datetime.timedelta(seconds=sleep)
```

Comment:
Even though this is fine, it does not fit with the consistency with the rest of the extension that is using the utc timezone.

Reply:
copy - what I have locally ensures everything has tz info.